lib/css/modules/_commentQuickCollapse.scss

```diff
@@ -0,0 +1,44 @@
+.res-commentQuickCollapse-toggleCommentsOnClickLeftEdge #siteTable ~ .commentarea {
+	.comment  {
+	/*
```

Comment:
Is this commented-out block useful? Since RES already has the commentBoxes styleTweaks, it feels superfluous here.